client/client.cpp

```diff
@@ -21,11 +20,24 @@ Operation genRandOperation() {
 	Operation op;
 	op.__set_op(r < 0.5 ? OpType::PUT : OpType::GET);
   op.__set_key(std::to_string(key));
-	if(op.op == OpType::GET) {
-		char value[VALUE_SIZE];
-		randombytes_buf(value, VALUE_SIZE);
-		op.__set_value(std::string(value));
+
+	std::string value;
+	if (op.op == OpType::GET) {
+		char rand_val[VALUE_SIZE];
+		randombytes_buf(rand_val, VALUE_SIZE);
+		value = std::string(rand_val);
+	} else {
+		int put_val = rand() % VAL_MAX;
+		value = std::to_string(put_val);
```

Comment:
I'm assuming this string stuff is modelled off sample implementations?

Reply:
The PUT case is currently a temporary solution where we generate a random value. Sample solutions have been using Proxy to populate value (or so I believe)... We should discuss this further with Sujaya tomorrow.

Reply:
I moreso meant are we expecting values to be strings

Reply:
Yep, all encryption and decryption take strings/char*